lib/elves.c

```diff
@@ -593,7 +591,7 @@ sr_elf_get_eh_frame(const char *filename,
         }
     }
 
-    if (-1 == exec_base)
+    if (exec_base == UINT64_MAX)
```

Comment:
Keeping rvalues on the left is generally not a bad idea, since accidentally dropping one of the equals signs will result in a compilation error.

Reply:
I understand the motivation; however, the style isn't consistent across the file, let alone the codebase, anyway, so I changed to what feels more natural to me. If it's a blocker, I will revert the order, but it seems unnecessary to me.